src/wdio/services/TerraService.js

```diff
@@ -37,21 +47,37 @@ function matchReference() {
 * @param sizes - [String] of viewport sizes.
 * @return [Object] of viewport sizes.
 */
-const viewports = (...sizes) => {
-  const widths = {
-    tiny: { width: 470, height: 768 },
-    small: { width: 622, height: 768 },
-    medium: { width: 838, height: 768 },
-    large: { width: 1000, height: 768 },
-    huge: { width: 1300, height: 768 },
-    enormous: { width: 1500, height: 768 },
-  };
+const getViewports = (...sizes) => {
+  let viewportSizes = Object.keys(VIEWPORTS);
+  if (sizes.length) {
+    viewportSizes = sizes;
+  }
+  return viewportSizes.map(size => VIEWPORTS[size]);
+};
+
+const beAccessible = (options) => {
+  global.it('is accessible', () => {
+    global.expect(global.browser.axe(options)).to.be.accessible();
+  });
+};
 
-  if (sizes.length === 0) {
-    return global.viewport('tiny', 'small', 'medium', 'large', 'huge');
+const matchScreenshot = (param1, param2) => {
+  let name = 'default';
+  let options = {};
+  if (typeof param1 === 'string') {
+    name = param1;
+    options = param2 || options;
+  } else {
+    options = param1 || options;
   }
 
-  return sizes.map(size => widths[size]);
+  const selector = options.selector || '[data-reactroot]';
+  const viewports = options.viewports || [];
+
+  global.it(`[${name}] matches screenshot`, () => {
```

Comment:
Is there a way to grab the test step name from this instance? that was how we originally proposed the naming structure
https://github.com/cerner/terra-toolkit/blob/master/tests/wdio/compare-spec.js#L7

Reply:
This is the test step.... this is a helper to inject it so it needs a name passed in

Reply:
This is the test step.... this is a helper to inject it so it needs a name passed in

Reply:
duh right. I was thinking in terms grabbing the describe for a test doing something like:

[1 / default] matches screenshot
[1 / default] displays modal currently on click
[1 / default] displays modal on exit

If not, I think we might want to change param1 to be test_step which does not have a default message.

Reply:
@ryanthemanuel Is this how you'd expected Orion testing to output for screenshots?

Reply:
Let me sync up with some folks on the Orion side to see if there's anything specific they're expecting.  I think we could still encapsulate the requirements inside of the name being passed in if necessary.

Reply:
```
Terra.should.matchScreenshot('345626, 3452534 Test that button responds accordingly', { selector: 'button', viewports });
```